### PR TITLE
add commit funnds sections

### PR DESCRIPTION
**Pull request by MostafaBalata**
screen shot 2017-08-15 at 6 17 20 pm

### CODE REVIEW

File: page/partials/commit_funds.ejs [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<div class="commit-funds section">
+        <div class="row">
+            <div class="col-md-6">            
+                <div class="number">
+                    &nbsp;&nbsp;&nbsp;&nbsp;01
```
Comment (krzkaczor) [inline page/partials/commit_funds.ejs]: wtf is that? xD I think using paddings to artificially expand element is a better idea.

File: page/sass/commit_funds.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.section.commit-funds{
+    h3{
```
Comment (krzkaczor) [inline page/sass/commit_funds.scss]: we are redefining font sizes like in ten places. I think we need to refactor it when all sections are done.

File: page/partials/commit_funds.ejs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                        deal flow and providing legal & technical 
+                        infrastructure. <strong>It does not manage 
+                        your funds for you.</strong>                   
+                    </p>
+                </div>
+            </div> <!-- end col-md-6 -->
+
+            <div class="col-md-6">
+                <div class="number">
+                    &nbsp;&nbsp;&nbsp;&nbsp;02
```
Comment (krzkaczor) [inline page/partials/commit_funds.ejs]: same here

File: page/sass/commit_funds.scss (hunk 2 above)
Comment (MostafaBalata) [inline page/sass/commit_funds.scss]: each section title has its own font-size, look at the design.

Comment (krzkaczor) [inline page/sass/commit_funds.scss]: I am sure that there are some similarity. If font is bigger/smaller just a little bit then we can assume its the same (mockups are not 100% correct). Lets see later what to do with that.

**Pull request merged**